smart-contracts/contracts/dex-router-osmosis/src/state.rs

```diff
@@ -0,0 +1,15 @@
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::Addr;
+use cw_storage_plus::{Item, Map};
+use mars_owner::Owner;
+use osmosis_std::types::osmosis::poolmanager::v1beta1::SwapAmountInRoute;
+
+#[cw_serde]
+pub struct RecipientInfo {
+    pub address: Addr,
+    pub denom: String,
+}
+
+pub const PATHS: Map<(String, String), Vec<Vec<SwapAmountInRoute>>> = Map::new("paths");
```

Comment:
Can we comment at line 12 what the index String, String represents?

Reply:
done